docker/jmeter-base/Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN curl -o /opt/apache-jmeter-$JMETER_VERSION.tgz \
     https://search.maven.org/remotecontent?filepath=kg/apc/cmdrunner/2.3/cmdrunner-2.3.jar \
  && curl -o /opt/apache-jmeter-$JMETER_VERSION/lib/postgresql-42.5.1.jar -L \
     https://jdbc.postgresql.org/download/postgresql-42.5.1.jar \
+ && rm /opt/apache-jmeter-$JMETER_VERSION/lib/mongo-java-driver-*.jar \
```

Comment:
_Context_: This is necessary to avoid compatibility issues between the default MongoDB Java Driver that comes with JMeter (e.g. `mongo-java-driver-2.11.3`) and the one that is being installed in the next line.